src/SingleBookingPage/bookingItem/BookingHeaders/Multicity.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import idx from 'idx';
+import { graphql, createFragmentContainer } from 'react-relay';
+
+import type { Multicity_bookingHeader } from './__generated__/Multicity_bookingHeader.graphql';
+
+type Props = {|
+  bookingHeader: Multicity_bookingHeader,
+|};
+
+const MulticityBookingHeader = ({ bookingHeader }: Props) => {
+  const trips = idx(bookingHeader, _ => _.trips) || [];
+  const destiantion = idx(bookingHeader.end, _ => _.airport.city.name) || '';
+  return (
+    trips.reduce((acc, trip) => {
+      const city = idx(trip, _ => _.departure.airport.city.name) || '';
+      return (acc += `${city} → `);
```

Comment:
Orbit components doesn't have available yet the `arrowRight` icon, and couldn't `transform: rotate(90deg)` the `arrowUp`, some ideas? or can we leave it like this until new Orbit update ?

Reply:
Why did the transform fail? 

We should not wait for orbit components forever, so I guess if there's no other way, we should leave it like this. I know FIXME comments are not exactly a great pattern but in that case maybe I would leave a FIXME comment there.

Reply:
I can rotate the `div` wrapping the icon, but not the icon itself, since is a component from Orbit I can't pass classes to it or use classes in it. Is a good suggestion FIXME comment here 👍, thanks.

Reply:
hmm is it possible to add an extra div around the icon and rotate that?
Although even if that works it might not be the most elegant solution long term :-D

Reply:
I tried it, without success 😞